_layouts/default.html

```diff
@@ -43,6 +43,16 @@ <h1><a href="{{ "/" | absolute_url }}">{{ site.title | default: site.github.repo
         {% include footer.html %}
       </footer>
     </div>
-    <script src="{{ "/assets/js/scale.fix.js" | relative_url }}"></script>
+<!-- Cookie Consent by TermsFeed https://www.TermsFeed.com -->
+<script type="text/javascript" src="https://www.termsfeed.com/public/cookie-consent/4.0.0/cookie-consent.js" charset="UTF-8"></script>
+<script type="text/javascript" charset="UTF-8">
+document.addEventListener('DOMContentLoaded', function () {
+cookieconsent.run({"notice_banner_type":"interstitial","consent_type":"express","palette":"light","language":"en","page_load_consent_levels":["strictly-necessary"],"notice_banner_reject_button_hide":false,"preferences_center_close_button_hide":false,"page_refresh_confirmation_buttons":false,"website_name":"code.gematik.de","website_privacy_policy_url":"https://www.gematik.de/datenschutz"});
```

Comment:
interstitial!??!

Reply:
would it be better to self-host the JavaScript?

Reply:
yea well it's a fast solution until Monday when we switch to the gematik Cookie platform via usercentrics